Sources/Disposable.swift

```diff
@@ -38,6 +38,13 @@ extension UnsafeAtomicState where State == DisposableState {
 	}
 }
 
+internal final class NopDisposable: Disposable {
+	static let shared = NopDisposable()
+	var isDisposed = false
+	func dispose() {}
+	private init() {}
+}
```

Comment:
This is a shared disposable for constant producers that cannot be disposed of. When I iterate on the design, benchmark shows allocation contributes considerable overhead.